app/build.gradle

```diff
@@ -1,5 +1,8 @@
 plugins {
```

Comment:
Converting the gradle scripts to kotlin will help too, Would you like me to handle it?

Reply:
~~Why do you think converting gradle scripts to kotlin is a good idea? Does it have a clearer syntax than Groovy?~~ [Did the research myself](https://stackoverflow.com/q/45335874/). Yes this is a good idea. Could you handle it, in a separate PR, please? I have no experience at all in Kotlin-for-gradle. ;-)